src/lib.rs

```diff
@@ -11,6 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#![cfg_attr(test, deny(warnings))]
```

Comment:
I think it would be confusing to contributors, why local tests passed but ci failed?